components/Cards/SchoolCard/__tests__/SchoolCard.test.js

```diff
@@ -5,33 +5,71 @@ import createSnapshotTest from 'test-utils/createSnapshotTest';
 
 import SchoolCard from '../SchoolCard';
 
+const mockManyLocationsWithGIAccepted = [
+  {
+    city: 'Austin',
+    state: 'TX',
+    doesAcceptGIBill: true,
+  },
+  {
+    city: 'Los Angeles',
+    state: 'CA',
+    doesAcceptGIBill: false,
+  },
+  {
+    city: 'New York',
+    state: 'NT',
+    doesAcceptGIBill: true,
+  },
+];
+
+// const mockManyLocationsWithoutGIAccepted = [
```

Comment:
Should code commented code be removed?

Reply:
I'm gonna use those later on for some tests to confirm business logic, such as the appearance of the text: "CITY, ST" given a single location.